src/test/ui/rust-2018/async-ident.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(dead_code, unused_variables, non_camel_case_types, non_upper_case_globals)]
+#![allow(dead_code, unused_variables, unused_macro_rules, bad_style)]
```

Comment:
Non-`run-pass` tests allow the whole `unused` lint group, so for them adding `allow(unused_macro_rules)` shouldn't be necessary.

Reply:
The issue is that this test also has rustfix enabled, and once that has ran, there are no warnings allowed. I've tried removing the `unused_macro_rules` allow again, it made the test fail.